Compile error on freebsd 12.2 on latest git pull.

i have all the Prerequisites as stated on ltfs page, i build and install like the list on page and get this:

see full run here: https://paste.sh/Hpr5W9zo#8_2dp1_bGaVBxMDP1aQO61pN
autogen
configure
make -> error down here.

Need recommandation on how to proceed to make this work? any hints.

```
~/ltfs # make
make  all-recursive
Making all in messages
Making all in src
Making all in libltfs
Making all in tape_drivers/generic/file
Making all in tape_drivers/generic/itdtimg
Making all in tape_drivers/freebsd/cam
/bin/sh ../../../../libtool  --tag=CC --tag=disable-static  --mode=compile cc -DHAVE_CONFIG_H -I. -I../../../..  -D_GNU_SOURCE -I../../../../src -DLTFS_CONFIG_FILE='"/usr/local/etc/ltfs.conf"' -DLTFS_BASE_DIR='"/usr/local"' -DUSE_NEW_LOCKING -I ../../.. -I ../..  -Wall -Wsign-compare -fsigned-char -I/usr/local/include/fuse -D_FILE_OFFSET_BITS=64 -I/usr/local/include/uuid -I/usr/local/include -I/usr/local/include/libxml2 -DU_DISABLE_RENAMING=1 -I/usr/local/include -D ENABLE_SNMP -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -DNETSNMP_ENABLE_IPV6 -fno-strict-aliasing -DNETSNMP_REMOVE_U64 -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -Ufreebsd12 -Dfreebsd12=freebsd12 -DHAS_FPSETMASK -DHAS_FLOATINGPOINT_H -DUSE_THREAD_SAFE_LOCALE -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -I/usr/local/lib/perl5/5.32/mach/CORE -I/usr/include -I. -I/usr/local/include -g -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -MT libtape_cam_la-cam_tc.lo -MD -MP -MF .deps/libtape_cam_la-cam_tc.Tpo -c -o libtape_cam_la-cam_tc.lo `test -f 'cam_tc.c' || echo './'`cam_tc.c
libtool: compile:  cc -DHAVE_CONFIG_H -I. -I../../../.. -D_GNU_SOURCE -I../../../../src -DLTFS_CONFIG_FILE=\"/usr/local/etc/ltfs.conf\" -DLTFS_BASE_DIR=\"/usr/local\" -DUSE_NEW_LOCKING -I ../../.. -I ../.. -Wall -Wsign-compare -fsigned-char -I/usr/local/include/fuse -D_FILE_OFFSET_BITS=64 -I/usr/local/include/uuid -I/usr/local/include -I/usr/local/include/libxml2 -DU_DISABLE_RENAMING=1 -I/usr/local/include -D ENABLE_SNMP -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -DNETSNMP_ENABLE_IPV6 -fno-strict-aliasing -DNETSNMP_REMOVE_U64 -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -Ufreebsd12 -Dfreebsd12=freebsd12 -DHAS_FPSETMASK -DHAS_FLOATINGPOINT_H -DUSE_THREAD_SAFE_LOCALE -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -I/usr/local/lib/perl5/5.32/mach/CORE -I/usr/include -I. -I/usr/local/include -g -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -MT libtape_cam_la-cam_tc.lo -MD -MP -MF .deps/libtape_cam_la-cam_tc.Tpo -c cam_tc.c  -fPIC -DPIC -o .libs/libtape_cam_la-cam_tc.o
cam_tc.c:286:4: error: use of undeclared identifier 'priv'
                        priv->vendor = (*cur)->vendor_type;
                        ^
1 error generated.
*** Error code 1

Stop.
make[3]: stopped in /root/ltfs/src/tape_drivers/freebsd/cam
*** Error code 1
*** Error code 1
*** Error code 1
```

--- a/src/tape_drivers/freebsd/cam/cam_tc.c
+++ b/src/tape_drivers/freebsd/cam/cam_tc.c
@@ -283,7 +283,7 @@ int camtape_open(const char *devname, void **handle)
 	while(*cur) {
 		if ((! strncmp((char*)softc->cd->inq_data.vendor, (*cur)->vendor_id, strlen((*cur)->vendor_id)) ) &&
 			(! strncmp((char*)softc->cd->inq_data.product, (*cur)->product_id, strlen((*cur)->product_id)) ) ) {
-			priv->vendor = (*cur)->vendor_type;
+			softc->vendor = (*cur)->vendor_type;
 			drive_type = (*cur)->drive_type;
 			break;
 		}

